src/main/java/com/thevoxelbox/voxelsniper/brush/type/CometBrush.java

```diff
@@ -12,35 +12,59 @@
 import org.bukkit.entity.SmallFireball;
 import org.bukkit.util.Vector;
 
+import java.util.List;
+import java.util.stream.Stream;
+
 public class CometBrush extends AbstractBrush {
 
     private boolean useBigBalls;
 
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
         SnipeMessenger messenger = snipe.createMessenger();
-        for (int index = 0; index < parameters.length; ++index) {
-            String parameter = parameters[index];
-            if (parameter.equalsIgnoreCase("info")) {
-                messenger.sendMessage("Parameters:");
-                messenger.sendMessage("balls [big|small]  -- Sets your ball size.");
-            }
-            if (parameter.equalsIgnoreCase("balls")) {
-                if (index + 1 >= parameters.length) {
-                    messenger.sendMessage("The balls parameter expects a ball size after it.");
-                }
-                String newBallSize = parameters[++index];
-                if (newBallSize.equalsIgnoreCase("big")) {
-                    this.useBigBalls = true;
-                    messenger.sendMessage("Your balls are " + ChatColor.DARK_RED + ("BIG"));
-                } else if (newBallSize.equalsIgnoreCase("small")) {
-                    this.useBigBalls = false;
-                    messenger.sendMessage("Your balls are " + ChatColor.DARK_RED + ("small"));
+        String firstParameter = parameters[0];
+
+        if (firstParameter.equalsIgnoreCase("info")) {
```

Comment:
Putting this here, if you set the comet brush, we could append a note that this brush is not undo-able due to the way it works but you can use `//ex` to extinguish the fire it creates.